diff --git a/node_modules/@theme-ui/css/dist/index.esm.js b/node_modules/@theme-ui/css/dist/index.esm.js
index 7dd4bad..755151a 100644
--- a/node_modules/@theme-ui/css/dist/index.esm.js
+++ b/node_modules/@theme-ui/css/dist/index.esm.js
@@ -133,7 +133,7 @@ var transforms = ['margin', 'marginTop', 'marginRight', 'marginBottom', 'marginL
 var responsive = function (styles) { return function (theme) {
   var next = {};
   var breakpoints = get(theme, 'breakpoints', defaultBreakpoints);
-  var mediaQueries = [null ].concat( breakpoints.map(function (n) { return ("@media screen and (min-width: " + n + ")"); }));
+  var mediaQueries = [null ].concat( breakpoints.map(function (n) { return ("@media (min-width: " + n + ")"); }));
 
   for (var key in styles) {
     var value = typeof styles[key] === 'function' ? styles[key](theme) : styles[key];
diff --git a/node_modules/@theme-ui/css/dist/index.js b/node_modules/@theme-ui/css/dist/index.js
index 31653f4..407ac78 100644
--- a/node_modules/@theme-ui/css/dist/index.js
+++ b/node_modules/@theme-ui/css/dist/index.js
@@ -133,7 +133,7 @@ var transforms = ['margin', 'marginTop', 'marginRight', 'marginBottom', 'marginL
 var responsive = function (styles) { return function (theme) {
   var next = {};
   var breakpoints = get(theme, 'breakpoints', defaultBreakpoints);
-  var mediaQueries = [null ].concat( breakpoints.map(function (n) { return ("@media screen and (min-width: " + n + ")"); }));
+  var mediaQueries = [null ].concat( breakpoints.map(function (n) { return ("@media (min-width: " + n + ")"); }));
 
   for (var key in styles) {
     var value = typeof styles[key] === 'function' ? styles[key](theme) : styles[key];
diff --git a/node_modules/@theme-ui/css/src/index.js b/node_modules/@theme-ui/css/src/index.js
index da56693..5339e6d 100644
--- a/node_modules/@theme-ui/css/src/index.js
+++ b/node_modules/@theme-ui/css/src/index.js
@@ -150,7 +150,7 @@ const responsive = styles => theme => {
   const breakpoints = get(theme, 'breakpoints', defaultBreakpoints)
   const mediaQueries = [
     null,
-    ...breakpoints.map(n => `@media screen and (min-width: ${n})`),
+    ...breakpoints.map(n => `@media (min-width: ${n})`),
   ]
 
   for (const key in styles) {
